src/modules/nfts/models/Nft.ts

```diff
@@ -6,12 +6,12 @@ import {
   Creator,
   MasterEditionV2Args,
 } from '@metaplex-foundation/mpl-token-metadata';
-import { Model } from '@/shared';
-import { MetadataAccount, MasterEditionAccount } from '@/programs/tokenMetadata';
-import { JsonMetadataLoader } from './JsonMetadataLoader';
-import { removeEmptyChars } from '@/utils';
-import { MasterEditionLoader } from './MasterEditionLoader';
-import { JsonMetadata } from './JsonMetadata';
+import { Model } from '../../../shared/index.js';
+import { MetadataAccount, MasterEditionAccount } from '../../../programs/tokenMetadata/index.js';
+import { JsonMetadataLoader } from './JsonMetadataLoader.js';
```

Comment:
It is super uncommon to mention the file extension `.js` in an import or `require`.
I don't understand why this is necessary now .. something seems off if it is.